content/projects/vgp/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: "Aiding in the creation a publicly-available database of genome assemblies of over 70,000 vertebrate species"
+autotoc: false
+title: "The Vertebrate Genomes Project in Galaxy"
+---
+
+<p align="center">
+  <img src="/images/vgp/vgp_logo.png"/>
+</p>
+
+## Vertebrate Genomes Project
+
+----
+
+The [Vertebrate Genomes Project](https://vertebrategenomesproject.org/) (VGP) is an international collaborative effort by the G10K consortium to generate near error-free genome assemblies for more than 70,000 vertebrate species. Using Galaxy infrasctructure and public instances, this collaboration has generated new, more open methods of genome assembly and access to data:
+ - Integration of the Genome Ark on public Galaxy servers.
+  - A Galaxy platform with toolkits specifically tailored for [Genome assembly](https://assembly.usegalaxy.eu).
+  - Multiple workflows available from the IWC on US and EU servers using the most up-to-date VGP pipelines.
+  - A list of publicly-available histories for each assembly completed on Galaxy as they are generated.
+
+
+<div class="row row-cols-1 row-cols-md-2">
+  <div class="col mb-4">
+    <div class="card h-100 shadow-sm bg-light rounded border-0">
+      <div class="card-body">
+        <h3><b>Data from GenomeArk</b></h3>
+        Galaxy has integrated the Genome Ark server as a remote data repository on all public servers. Using the remote data importer, the initial sequence files for all available species as well as the completed assemblies for species as they are completed are available to any logged-in user.<br><br>
+        <a href="https://vgp.github.io/genomeark/" class="btn btn-primary">Explore GenomeArk</a>
+      </div>
+    </div>
+  </div>
+  <div class="col mb-4">
+    <div class="card h-100 shadow-sm bg-light rounded border-0">
+      <div class="card-body">
+        <h3><b>Analysis Platform</b></h3>
+        All tools and workflows are publicly available and continuously on the assembly-specific Galaxy workbench, along with relevant links to trainings, workflows, and example analyses. <br><br>
```

Comment:
All tools and workflows are publicly available through Galaxy workbench, along with relevant links to trainings and example analyses.